backend/console/resources/public/static/openapi.yaml

```
@@ -5,8 +5,127 @@
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/Response"
+                  - $ref: "#/components/schemas/HTTPCreatedCode"
+                  - type: object
+                    properties:
+                      data:
+                        type: object
+                        properties:
+                      data:
```

YAMLlint (1.35.1)

[error] duplication of key "data" in mapping

(key-duplicates)